client/components/AttributionEditor/EditableAvatar.tsx

```diff
@@ -9,7 +9,7 @@ require('./editableAvatar.scss');
 
 type Props = {
 	attribution: {
-		avatar?: string;
+		avatar?: string | null;
```

Comment:
This is how i general tried to solve the `string | null cannot be assigned to type string | undefined. null cannot be assigned to undefined` errors. In most cases this is enough, sometimes I need to add some extra `avatar ?? undefined` somewhere in the component bc e.g. blueprint does not accept null